libtestutils/Common.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+	This file is part of cpp-ethereum.
+ 
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+ 
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+ 
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/** @file Common.cpp
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ */
+
+#include <random>
+#include <libdevcore/CommonData.h>
+#include <libdevcore/CommonIO.h>
+#include <libdevcrypto/FileSystem.h>
+#include "Common.h"
+
+using namespace std;
+using namespace dev;
+using namespace dev::test;
+
+bytes dev::test::toBytes(std::string const& _str)
+{
+	return fromHex(_str.substr(0, 2) == "0x" ? _str.substr(2) : _str, ThrowType::Throw);
+}
+
+std::string dev::test::getTestPath()
```

Comment:
that is a duplication of https://github.com/ethereum/cpp-ethereum/blob/develop/test/TestHelper.cpp#L377.
Why dont you use this function directly?